src/App.js

```diff
@@ -1,13 +1,13 @@
 import logo from './logo.svg';
 import './App.css';
 
-import GPS from './components/gps/GPS';
+import GoeWebSystem from './components/GeoWebSystem/GeoWebSystem';
```

Comment:
Looks like a small typo here.

Reply:
`GoeWebSystem` -> `GeoWebSystem`

Reply:
Resolving it the next request. Alternatively, thinking of naming the component GWS (on the lines of GPS)